pyfolio/plotting.py

```diff
@@ -641,7 +649,25 @@ def plot_rolling_returns(
             ax.fill_between(cone_df_live.index,
                             cone_df_live.sd_down,
                             cone_df_live.sd_up,
-                            color='red', alpha=0.30)
+                            color='steelblue', alpha=0.25)
+
+            if type(cone_std) is list:
```

Comment:
cool, yea the appropriate way to do type checking in Python seems to be a pretty controversial issue on stackoverflow :)  

**Sent from iPhone**




On Tue, Oct 13, 2015 at 6:51 AM -0700, "Thomas Wiecki" <notifications@github.com> wrote:












In pyfolio/plotting.py:
> @@ -641,7 +649,25 @@ def plot_rolling_returns(
>              ax.fill_between(cone_df_live.index,
>                              cone_df_live.sd_down,
>                              cone_df_live.sd_up,
> -                            color='red', alpha=0.30)
> +                            color='steelblue', alpha=0.25)
> +
> +            if type(cone_std) is list:



There might be a more elegant way to do this. I'll think abit on it.



—
Reply to this email directly or view it on GitHub.